jekyll/_cci2/api-developers-guide.md

```diff
@@ -450,6 +450,184 @@ The following section details the steps you need to follow to download artifacts
 
     **Note:** `grep` is used to locate all the URLs for downloading the job artifacts, while `wget` is used to perform the download.
 
+## Gather Insights
+
+The CircleCI API v2 also includes several endpoints that enable you to retrieve detailed insights into your workflows and individual builds. By making API calls to these endpoints, you can better understand how to optimize your workflows and builds so you can increase workflow performance while minimizing credit usage and consumption. The example below describes how you can return information about a single workflow containg informaiton about metrics and credit usage.
+
+### Returning Workflow Metrics
+
+To return aggregated data for an individual workflow, perform the steps listed below.
+
+**Note:** whenever you see curly brackets `{}`, this represents a variable that you must manually enter in the request.
+
+1. Declare the parameters you expect to receive from the API. For this GET API call, under the `parameters` key, define the `project_slug` in your `curl` request as follows:
```

Comment:
I don't get the reference to the `parameters` key here, is it not just "form your API GET request as shown below" or something? I am no expert though of course! (same in the get job details use cases)